lib/RateLimiterQueue.js

```diff
@@ -3,14 +3,14 @@ const MAX_QUEUE_SIZE = 4294967295;
 const KEY_DEFAULT = 'limiter';
 
 module.exports = class RateLimiterQueue {
-  constructor(limiterFlexible, opts = {
-    maxQueueSize: MAX_QUEUE_SIZE,
-  }) {
+  constructor(limiterFlexible, opts = {}) {
+    const maxQueueSize = opts.maxQueueSize !== undefined ? opts.maxQueueSize : MAX_QUEUE_SIZE;
+    const key = opts.key !== undefined ? opts.key : KEY_DEFAULT;
```

Comment:
@sevauni Could you please explain how do you use the new `key` option?

Reply:
That makes sense, thank you.
`key` attribute should be kept internal.
It should be always set to `KEY_DEFAULT` value in RateLimiterQueue constructor.